docs/operators/how-to/configure-rest-api.md

```diff
@@ -0,0 +1,20 @@
+# Node configuration
```

Comment:
Thanks for including operator documentation in the PR! It's important to keep this up to date.

In general, we want every new page in the operator guide to be reachable from the landing page with a few clicks. In other words, it will be useful if this page is linked from somewhere else - I'd suggest just adding this to the configuration tutorials list in the [configuration landing page](https://github.com/waku-org/nwaku/blob/0627b4f8f2c83d34ece6baeaa64b2d8b9244937b/docs/operators/how-to/configure.md#configuration-use-cases)